installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,112 @@
+[id="manually-creating-iam"]
+= Manually creating IAM
+include::modules/common-attributes.adoc[]
+:context: manually-creating-iam
+
+toc::[]
+
+If you do not allow the system to manage identity and access management (IAM),
+then you can manually create IAM credentials.
+
+In environments where the cloud IAM APIs are not reachable, the Cloud Credential
+Operator can be disabled prior to installation.
+
+.Procedure
+
+. Run the {product-title} installer to generate manifests:
++
+----
+$ openshift-install create manifests --dir=mycluster
+----
+
+. Insert a ConfigMap into the manifests directory to disable the Cloud Credential Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+----
+
+. Identify all `CredentialsRequests` in your release image that target the cloud
+you are deploying on. You must manually create credentials that provide the
+permissions in each of these requests, base64 encode them, and store them in
+secrets with the name and namespace defined in the `CredentialsRequest` target
+secret.
++
+It might be helpful to extract these locally:
++
+----
+$ oc adm release extract registry.svc.ci.openshift.org/ocp/release:4.3.0-0.ci-2019-10-23-103858
+----
++
+Then, search for `CredentialsRequest`.
+
+. Create credentials in AWS with permissions matching each `CredentialsRequest`.
+It is possible, though not ideal, to use one set of credentials for all
+components.
+
+. Create the required secrets as installer manifests. You do not want to hard code
+credentials into a published release image. Add a Secret YAML file to the
+cluster's manifests directory for each `CredentialsRequest` identified. Each
+access key ID and secret access key should be base64 encoded.
```

Comment:
If you think it might be better, we could skip the base64 encoding and use stringData instead of data, a convenience Kube makes available to b64encode for you when creating only. (won't be present if you view the yaml/json later) It might simplify the docs a little bit.